app/logic/transcript.py

```diff
@@ -47,8 +56,14 @@ def getTotalHours(username):
     """
     Get the toal hours from events and courses combined.
     """
+    transcriptsRemoved = [program.program_id for program in ProgramBan.select().where((ProgramBan.user == username) and (ProgramBan.unbanNote.is_null(True)) and (ProgramBan.removeFromTranscript == 1))]
```

Comment:
Let's pull the query out into a variable so that the query can be formatted more nicely and our list comprehension is a little more readable.